larrecodnn/CVN/interfaces/PixelMapProducer.h

```diff
@@ -86,8 +86,19 @@ namespace lcvn {
     // overload constructor for inputs from fcl
     PixelMapProducer(const fhicl::ParameterSet& pset);
 
-    void SetMultipleDrifts() { fMultipleDrifts = true; }
-    unsigned int NROI() { return fTotHits; };
+    unsigned int NWire() const { return fNWire; }
+    unsigned int NTdc() const { return fNTdc; }
+    double TRes() const { return fTRes; }
+    unsigned int TotHits() const { return fTotHits; }
+    double Threshold() const { return fThreshold; }
+    bool MultipleDrifts() const { return fMultipleDrifts; }
+
+    void SetNWire(unsigned int nwire) { fNWire = nwire; }
+    void SetNTdc(unsigned int ntdc) { fNTdc = ntdc; }
+    void SetTRes(double tres) { fTRes = tres; }
+    void SetTotHits(unsigned int tothits) {fTotHits = tothits; }
+    void SetThreshold(double threshold) {fThreshold = threshold; }
```

Comment:
Ideally, there would be no setters as that can cause multithreading issues.  However, getting there would take some re-architecting of the code.  In this case, I suggest implementing only the setters that you need.

Reply:
Thanks @knoepfel, I have removed unnecessary setter functions.